src/classes/trackers.js

```diff
@@ -34,6 +34,7 @@
  * @typedef OwnerData
  * @property {string} name
  * @property {string} displayName
+ * @property {string | undefined} ownedBy
```

Comment:
Nit you can use which the file uses:
```suggestion
 * @property {string} [ownedBy]
```

Reply:
Thanks, TIL a new jsdoc syntax.